packages/frontend-2/lib/settings/graphql/queries.ts

```diff
@@ -30,3 +30,14 @@ export const settingsWorkspacesMembersQuery = graphql(`
     }
   }
 `)
+
+export const settingsUserEmailsQuery = graphql(`
+  query SettingsUserEmailsQuery {
+    activeUser {
```

Comment:
All of the ActiveUser fields should be its own `SettingsUserEmail_User` fragment that you can then reference in all of the mutations, instead of duplicating `id` and `emails` every time

Reply:
Makes sense, I've updated it :) Everything works with the cache, didn't run into any issues when testing